src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1456,6 +1459,20 @@ BlittableJsonReaderObject UpdateDatabaseRecordIfNeeded()
             }
         }
 
+        private static bool IsExcluded(BlittableJsonReaderObject configurationBlittable, string databaseName)
+        {
+            if (configurationBlittable.TryGet(nameof(ServerWideBackupConfiguration.DatabasesToExclude), out BlittableJsonReaderArray databasesToExclude) == false)
+                return false;
+
+            foreach (object databaseToExclude in databasesToExclude)
+            {
+                if (databaseName.Equals(databaseToExclude.ToString(), StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
                if (string.Equals(databaseName, databaseToExclude.ToString(), StringComparison.OrdinalIgnoreCase))
```

(Avoids potential NRE)

Reply:
done